apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -156,7 +156,7 @@ export default async function (server: any, database: any): Promise<DatabaseClie
 
   const version = await getVersion(conn)
 
-  const features = version.isRedshift ? { customRoutines: true, comments: false, properties: false } : { customRoutines: true, comments: true, properties: true}
+  const features = version.isRedshift ? { customRoutines: true, comments: false, properties: false, partitions: false } : { customRoutines: true, comments: true, properties: true, partitions: true}
```

Comment:
Should probably check to see if partitions are supported on for the version of Postgres running, as it was only introduced in version 10 I believe.